.docforge/guides.yaml

```diff
@@ -1,5 +1,3 @@
-nodesSelector:
```

Comment:
why don't we use the nodesSelector instead of defining a custom structure which will look in the same way at the end?

Reply:
If the nodeSlector is on the top-level path, the following node collisions are visible:
```
In docs.guides container node.
  Node with name install_gardener appears 2 times for sources: , . 
  Node with name administer_shoots appears 2 times for sources: , . 
  Node with name applications appears 2 times for sources: , .
```
To resolve these conflicts, a separate node selector is defined for each sublevel. 
Once the node merging functionality is ready, such node collisions will not occur and the nodeSelector can be returned back to the top-level.

Reply:
I propose to remove the older (obsolete) versions of the duplicated pages

Reply:
Yes I also think that we should remove them but also in my opinion, the newer versions of the pages should be moved to https://github.com/gardener/documentation/tree/master/website/documentation/guides

In this way we will have just one nodeSelector which will selects all of the up-to-date pages.